.gitignore

```diff
@@ -0,0 +1 @@
+/nbproject
```

Comment:
then I will need to exclude nbproject folder every time I make a PR... not sure why to remove it, this is only developer thing :smile: